glib/src/param_spec.rs

```diff
@@ -898,6 +1079,32 @@ impl ParamSpecOverride {
             from_glib_none((*ptr).overridden)
         }
     }
+
+    pub fn builder<'a>(name: &'a str, overridden: &'a ParamSpec) -> ParamSpecOverrideBuilder<'a> {
+        ParamSpecOverrideBuilder::new(name, overridden)
+    }
+}
+
+// This builder is not autogenerated because it's the only one that doesn't take
+// `nick`, `blurb` and `flags` as parameters.
+#[must_use]
+pub struct ParamSpecOverrideBuilder<'a> {
```

Comment:
ParamSpec has two useful constructors that what people will use almost everytime (that takes either an interface/or class) to override it property. Do we need that integrated somehow as part of the builder?

Reply:
I don't know how to add the two constructors and make everything usable from the `Props` macro.

Reply:
This specific pspec can get some special handling in the macro. It's a very special case that doesn't appear anywhere else, and it's a "basic" pspec. You can invent some special syntax for this one if that's necessary.

Reply:
What I meant here is whether such builder is really useful to have at all

Reply:
As mentioned by slomo we should just sepcial case this one in the proc macro

Reply:
Reasonable